Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/IInputSource.cs

```diff
@@ -2,10 +2,9 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Collections;
-using Microsoft.MixedReality.Toolkit.InputSystem.Pointers;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions;
 
-namespace Microsoft.MixedReality.Toolkit.InputSystem.Sources
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
```

Comment:
MRTK Interface names should be prefixed with either IMixedReality or IMixedRealityToolkit

Reply:
lol I got it the first time.